app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/PTEnrolmentOnAnotherAccount.scala.html

```diff
@@ -0,0 +1,66 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.routes
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.AccountDetails
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.config.AppConfig
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+
+@this(
+layout: Layout,
+govukSummaryList : GovukSummaryList)
+
+@(ptAccountDetails: AccountDetails, currentAccountHasSA: Boolean)(implicit request: Request[AnyRef], messages: Messages, appConfig: AppConfig)
+
+@createSummaryListRow(key: String, value: String) =  {
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        @key
+      </dt>
+      <dd class="govuk-summary-list__value">
+        @value
+      </dd>
+    </div>
+}
+
+@layout(
+    pageTitle = Some(messages("ptEnrolmentOnOtherAccount.title")), None) {
+
+    <h2 class="govuk-heading-xl">@messages("ptEnrolmentOnOtherAccount.heading")</h2>
+    <p class="govuk-body">@messages("ptEnrolmentOnOtherAccount.text1")
+        <a href="@routes.SignOutController.signOut">@messages("ptEnrolmentOnOtherAccount.signinAgain")</a>
+      @messages("ptEnrolmentOnOtherAccount.text2")
+    </p>
+    <dl class="govuk-summary-list">
+        @createSummaryListRow(messages("ptEnrolmentOnOtherAccount.userId"), ptAccountDetails.userId)
+        @if(ptAccountDetails.email.isDefined) {
+          @createSummaryListRow(messages("ptEnrolmentOnOtherAccount.email"), ptAccountDetails.email.get)
+        }
```

Comment:
Last signed in isn't being displayed on the page, was this intentional? It appears in the prototype.